src/assets/js/protocol/protocol-newsletter.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+    'use strict';
+
+    // !! This file assumes only one signup form per page !!
+
+    // Expand email form on input focus or submit if details aren't visible
+    function initEmailForm() {
+        var newsletterForm = document.getElementById('newsletter-form');
+        var submitButton = document.getElementById('newsletter-submit');
+        var formDetails = document.getElementById('newsletter-details');
+        var formControls = document.querySelectorAll('.mzp-c-newsletter-form input, .mzp-c-newsletter-form select');
+        var formExpanded;
+
+        if (window.getComputedStyle(formDetails).display === 'none') {
+            formExpanded = false;
+        }
+
+        function emailFormShowDetails() {
+            if (!formExpanded) {
+                formDetails.style.display = 'block';
+                formExpanded = true;
+            }
+        }
+
+        for (var i = 0; i < formControls.length; i++) {
+            formControls[i].addEventListener('focus', function() {
+                emailFormShowDetails();
```

Comment:
Do we need to add `if (!formExpanded) { }` here?

Reply:
The `emailFormShowDetails` function already checks for it, but we need to check if the form is expanded on the submit button just for the sake of `e.preventDefault`. I'm sure there's a more elegant way to do this... I'm not good at JavaScript.